frontend/lib/src/theme/globalStyles.ts

```diff
@@ -313,109 +240,6 @@ export const globalStyles = (theme: EmotionTheme): SerializedStyles => css`
     border-style: none;
   }
 
-  // 1. Textareas should really only resize vertically so they don't break their (horizontal) containers.
-
-  textarea {
-    resize: vertical; // 1
```

Comment:
this is set on the BaseUI element, right?

Reply:
yep 👍